Frameworks/CoreFoundation/NumberDate.subproj/CFTimeZone.c

```diff
@@ -53,26 +55,12 @@
 #include <Foundation/NSTimeZone.h>
 #include <Foundation/NSDate.h>
 
-struct tzhead {
```

Comment:
I think we should have a single one at the top that explains the scope of the changes made. just something so that if/when we merge, people understand that basically this is a new file....

---
In reply to: [72719052](https://github.com/Microsoft/WinObjC/pull/701#discussion_r72719052) [](ancestors = 72719052,72719013,72718150)